fission/src/ui/panels/mirabuf/ImportMirabufPanel.tsx

```diff
@@ -82,47 +82,103 @@ const ItemCard: React.FC<ItemCardProps> = ({ id, name, primaryButtonNode, primar
 export type MiraManifest = {
     robots: MirabufRemoteInfo[]
     fields: MirabufRemoteInfo[]
+    pieces: MirabufRemoteInfo[]
 }
 
 function getCacheInfo(miraType: MiraType): MirabufCacheInfo[] {
-    return Object.values(
-        canOPFS
-            ? MirabufCachingService.getCacheMap(miraType)
-            : miraType === MiraType.ROBOT
-              ? backUpRobots
-              : backUpFields
-    )
+    return Object.values(canOPFS ? MirabufCachingService.getCacheMap(miraType) : backUpMap[miraType])
 }
 
+// TODO Move this function to lib file since we can't export non-components with components
```

Comment:
This PR todo or something else? This seems like something that should be done elsewhere?